runtime/src/parachains.rs

```diff
@@ -206,6 +218,9 @@ const WATERMARK_QUEUE_SIZE: usize = 20000;
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Parachains {
+		/// All authorities' keys at the moment.
+		pub Authorities get(authorities) config(authorities): Vec<ParachainPublic>;
+
 		// Vector of all parachain IDs.
 		pub Parachains get(active_parachains): Vec<ParaId>;
 		// The parachains registered at present.
```

Comment:
```suggestion
		/// The parachains registered at present.
```

Reply:
change to triple slash instead of double slash for other comments below

Reply:
- add newline above comment for `RelayDispatchQueueSize`
- remove the excessive spacing in `//	second if the total length (in bytes) of the message payloads.`